README.md

```diff
@@ -2,6 +2,12 @@
 <img src='logo/logo.svg' width='150px' align="right" style="float:right;margin-left:10pt"></img>
 Comple**X** **G**roup **I**nteractions (**XGI**) is a Python package for the representation, manipulation, and study of the structure, dynamics, and functions of complex systems with group (higher-order) interactions.
 
+[![Test status](https://github.com/ComplexGroupInteractions/xgi/workflows/test/badge.svg?branch=main)](https://github.com/ComplexGroupInteractions/xgi/actions?query=workflow%3A%22test%22)
```

Comment:
The badges appear on different lines, I'd put them on a single line?

Reply:
sounds good!